config-linux.md

```diff
@@ -158,13 +158,20 @@ cgroups provide controls to restrict cpu, memory, IO, pids and network for the c
 For more information, see the [kernel cgroups documentation][cgroup-v1].
 
 The path to the cgroups can be specified in the Spec via `cgroupsPath`.
-`cgroupsPath` is expected to be relative to the cgroups mount point.
-If `cgroupsPath` is not specified, implementations can define the default cgroup path.
+If `cgroupsPath` is
+* ... an absolute path (starting with `/`), the implementation MUST take the path to be relative to the cgroup mount point.
+* ... a relative path (not starting with `/`), implementations SHOULD take the value to be the suffix of the resulting cgroup path.
```

Comment:
On Wed, Jun 08, 2016 at 10:07:28PM -0700, Aleksa Sarai wrote:
> > +* ... a relative path (not starting with `/`), implementations SHOULD take the value to be the suffix of the resulting cgroup path.
>
> I'm trying to make it not-too restrictive. There's no reason that a
> runtime has to make it relative to the cgroups of the runtime
> process. "Should be the suffix" means that it should be the last
> component in the path (but component is the wrong word since it can
> container `/`).

How about:

  … MAY interpret the path relative to a runtime-determined location
  in the cgroup hierarchy.